### ISSUE TITLE
Using argument nameMapper in function runOrExit does not compile

### ISSUE THREAD

**Issue by jk-1** [com-lihaoyi/mainargs#154]
Mainargs docs Customization says:

`nameMapper: String => Option[String]`: how Scala camelCase names are mapping to CLI command and flag names. Defaults to translation to kebab-case, but you can pass in `mainargs.Util.snakeCaseNameMapper` for snake_case CLI names or `mainargs.Util.nullNameMapper` to disable mapping.

I tried this code: 

`  def main(args: Array[String]): Unit = ParserForMethods(this).runOrExit(args, allowPositional = true, nameMapper = mainargs.Util.snakeCaseNameMapper)
`
and got the error:

```
scala-cli MyTool.scala -- --help
Compiling project (Scala 2.13.14, JVM (17))
[error] ./MyTool.scala:948:104
[error] unknown parameter name: nameMapper
[error]   def main(args: Array[String]): Unit = ParserForMethods(this).runOrExit(args, allowPositional = true, nameMapper = mainargs.Util.snakeCaseNameMapper)
[error]                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error compiling project (Scala 2.13.14, JVM (17))
Compilation failed

```
Mainargs code is

```
def runOrExit(
      args: Seq[String],
      allowPositional: Boolean = false,
      allowRepeats: Boolean = false,
      stderr: PrintStream = System.err,
      totalWidth: Int = 100,
      printHelpOnExit: Boolean = true,
      docsOnNewLine: Boolean = false,
      autoPrintHelpAndExit: Option[(Int, PrintStream)] = Some((0, System.out)),
      customNames: Map[String, String] = Map(),
      customDocs: Map[String, String] = Map()
  ): Any

```  
So there is no nameMapper.

Is this error in mainargs code/docs or do I use it somehow incorrectly?
  
Used mainargs v is 0.7.2

**Comment by lihaoyi** [comment]
Looks like we added it to all the `runOr*` methods except forgetting about `runOrExit`

**Comment by lihaoyi** [comment]
https://github.com/com-lihaoyi/mainargs/pull/155

**Comment by jk-1** [comment]
https://github.com/com-lihaoyi/mainargs/blob/main/mainargs/src/Parser.scala

I found it only in runRaw, no other run* funcs.

**Comment by lihaoyi** [comment]
runEither has it https://github.com/com-lihaoyi/mainargs/blob/main/mainargs/src/Parser.scala#L139

**Comment by lihaoyi** [comment]
runOrThrow too https://github.com/com-lihaoyi/mainargs/blob/main/mainargs/src/Parser.scala#L109

**Comment by lihaoyi** [comment]
Should be fixed in 0.7.3 which is on its way to Maven Central
